steelthread-0/walkthrough.md

```diff
@@ -25,62 +28,68 @@ You should also take the opportunity to login your oc CLI as this user. You can
 
 # Setting up a Enmasse queue in your namespace 
 
+- From your namespace click the catalog button on the left hand nav
 - Provision a Enmasse service from the catalog (choose EnMasse standard) name it eval and choose not to bind at this time.
 - Once complete, in the Eval namespace you will find a EnMasse provisioned service.
-- Create a binding to this provisioned service checking each available option.
+- Now create a binding to this provisioned service checking each available option.
 - Collapse the provisioned service and click on the dashboard link which will show on the right hand side of the EnMasse provisioned service. This link will bring you to the EnMasse dashboard. 
 - When asked to login click the OpenShift button and login with the same user that provisioned the service. In an eval environment this will be ```evals@example.com```
 - Create a new address in the EnMasse console named ```work-queue/requests``` choose the ```queue``` option and the pooled plan.
-- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the multicast option and the standard plan
+- Create a second address in the EnMasse console named ```work-queue/worker-updates``` choose the ```queue``` option and the standard plan
```

Comment:
We mention standard plan here, should it not be pooled plan?

Reply:
Looks like we're using the term that's shown in the Service Catalog UI

![image](https://user-images.githubusercontent.com/878251/44520534-03aadf00-a6c8-11e8-913b-2276f38ace9f.png)